content/src/main/content/jcr_root/apps/acs-tools/components/dumplibs/clientlibs/js/components/crx-link.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * ACS AEM Tools Package
+ * %%
+ * Copyright (C) 2014 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+/*global Vue, axios, console */
+(function () {
+    'use strict';
+    Vue.component('crx-link', {
+        props: ['link'],
+        methods: {
+            isLink: function (l) {
+
+                return Vue.type.isString(l) && l.startsWith('/');
+            },
+            crxLink: function (link) {
+                return '/crx/de/index.jsp#' + link;
```

Comment:
Same as above wrt to servlet context path ... idk if this is a total PITA or easy.